src/FluentMigrator.Runner.Firebird/Generators/Firebird/FirebirdGenerator.cs

```diff
@@ -39,23 +40,26 @@ public class FirebirdGenerator : GenericGenerator
         protected readonly FirebirdTruncator truncator;
 
         public FirebirdGenerator(
-            [NotNull] FirebirdOptions fbOptions)
-            : this(fbOptions, new OptionsWrapper<GeneratorOptions>(new GeneratorOptions()))
+            [NotNull] FirebirdOptions fbOptions,
+            [NotNull] IOptions<QuoterOptions> quoterOptions)
+            : this(fbOptions, quoterOptions, new OptionsWrapper<GeneratorOptions>(new GeneratorOptions()))
         {
         }
 
         public FirebirdGenerator(
             [NotNull] FirebirdOptions fbOptions,
+            [NotNull] IOptions<QuoterOptions> quoterOptions,
             [NotNull] IOptions<GeneratorOptions> generatorOptions)
-            : this(new FirebirdQuoter(fbOptions.ForceQuote), fbOptions, generatorOptions)
+            : this(new FirebirdQuoter(fbOptions, quoterOptions), fbOptions, quoterOptions, generatorOptions)
         {
         }
 
         public FirebirdGenerator(
```

Comment:
We need an obsolete overload for this constructor unfortunately.  (Unless I am missing something.)